source_test.yaml

```diff
@@ -191,6 +203,18 @@
   link: 'https://github.com/ossf/malicious-packages/blob/main/'
   editable: False
 
+- name: 'test-oss-fuzz'
```

Comment:
oss-fuzz won't work for test because the necessary creds aren't there. OSS-Fuzz is also a rare source where OSV pushes changes, so we definitely can't have both prod and test do this.

Reply:
This was purely a reordering of what was already present, so should this be removed entirely?

Reply:
Ah, I see that `editable` is `False` here, which is what controls the write. Let's leave this in if this was just a reorder.